packages/warehouses/src/warehouseClients/TrinoWarehouseClient.ts

```diff
@@ -331,15 +331,41 @@ export class TrinoWarehouseClient extends WarehouseBaseClient<CreateTrinoCredent
                    data_type
             FROM information_schema.columns
             WHERE table_name = '${this.sanitizeInput(tableName)}'
-            ${schema ? `AND table_schema = ${this.sanitizeInput(schema)}` : ''}
+            ${
+                schema
+                    ? `AND table_schema = '${this.sanitizeInput(schema)}'`
+                    : ''
+            }
             ${
                 database
-                    ? `AND table_catalog = ${this.sanitizeInput(database)}`
+                    ? `AND table_catalog = '${this.sanitizeInput(database)}'`
                     : ''
-            };
+            }
         `;
         const { rows } = await this.runQuery(query, tags);
 
         return this.parseWarehouseCatalog(rows, convertDataTypeToDimensionType);
     }
+
+    async getAllTables() {
+        const databaseName = this.connectionOptions.catalog;
+        const whereSql = databaseName
+            ? `AND table_catalog = '${this.sanitizeInput(databaseName)}'`
```

Comment:
trino client doesn't take arguments on the query (we can't use ?) so afaik the only way is to sanitize the input on the sql